src/python/ensembl/io/genomio/gff3/process_gff3.py

```diff
@@ -941,12 +938,11 @@ def main() -> None:
     # Merge multiline gene features in a separate file
     interim_gff_path = Path(f"{in_gff_path}_INTERIM_MERGE")
     merger = GFFGeneMerger()
-    num_merged_genes = merger.merge(in_gff_path, interim_gff_path)
-
-    # If there are split genes, decide to merge, or just die
+    merged_genes = merger.merge(in_gff_path, interim_gff_path)
+    num_merged_genes = len(merged_genes)
     if num_merged_genes > 0:
-        if mod.args["retain_split_genes"]:
-            raise GFFParserError("GFF contains split genes. Fix it or remove '--retain_split_genes' flag.")
+        # Report the list of merged genes in case something does not look right
+        print(f"{num_merged_genes} genes merged:\n" + "\n".join(merged_genes))
```

Comment:
Should we output a log file with the GCA to store the merge gene event?

Reply:
At the moment we have no logging system in place, so a comprehensive print is the best we can do. Since this is stored in Nextflow's log file, and it is one per job, I guess it is ok. But feel free to correct me if I'm wrong.